scripts/util/detect-changed-contracts.js

```diff
@@ -100,7 +106,9 @@ async function detectChangedContract(referenceCommit, targetCommit) {
   console.log(`Total changed contracts: ${changedContracts.length}`);
 
   // Checkout back to original branch
+  shell.exec(`rm -rf contracts/*`);
```

Comment:
Doesn't need the glob, but if leaving in, should add into line 53 which does the same without the glob.

Reply:
Fixed.